CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v0.11.0 (November 25, 2022)
+Added feature in order to prevent an issuance of the certificate if it is already inside Vault storage, using a hash string generated. All operation are done locally.
```

Comment:
I think is better to say something like "Improved the feature to prevent the issuance of a certificate if it exists in Vault storafe already."